operator/internal/manifests/openshift/options.go

```diff
@@ -75,27 +89,22 @@ func NewOptions(
 		authn = append(authn, AuthenticationSpec{
```

Comment:
```suggestion
		o.Authentication = append(o.Authentication, AuthenticationSpec{
```

Reply:
You can directly use `o.Authentication`

Reply:
Strictly speaking, this is only true if we know that `o.Authentication` is an empty slice before, otherwise this leads to a different behaviour. This function is used in a sort-of-fluent-interface kind of way. Usually when you do that you directly chain the function calls together, which we are currently not doing.

I think I'd rather leave the definitions of `authz`/`authn` the way they currently are, so that the "output" of this function is completely constructed inside of it.

Reply:
That's right, I thought this was a call once and done function.